tests/php/sync/test_class.jetpack-sync-plugins.php

```diff
@@ -61,24 +61,24 @@ public function test_installing_and_removing_plugin_is_synced() {
 
 	public function test_autoupdate_enabled_and_disabled_is_synced() {
 		// enable autoupdates
-		$autoupdate_plugins = Jetpack_Options::get_option( 'autoupdate_plugins', array() );
+		$autoupdate_plugins = (array) get_site_option( 'auto_update_plugins', array() );
 		$autoupdate_plugins = array_unique( array_merge( $autoupdate_plugins, array( 'hello' ) ) );
-		Jetpack_Options::update_option( 'autoupdate_plugins', $autoupdate_plugins );
+		update_site_option( 'auto_update_plugins', $autoupdate_plugins );
 		$this->sender->do_sync();
```

Comment:
As this option isn't part of the always send "critical" options we need to make an explicit call to unlock callables sync. 
`do_action( 'jetpack_sync_unlock_sync_callable' );`

Reply:
if we do want to make this a critical option then we would need to make a few changes to class-callables.php in sync/modules

1) add "auto_update_plugins" to ALWAYS_SEND_UPDATES_TO_THESE_OPTIONS const
2) add "'auto_update_plugins' => 'autoupdate_plugins'," to OPTION_NAMES_TO_CALLABLE_NAMES
3) within init_listeners, we would need to update the foreach to include hooks to 'update_site_option_{$option}' and 'delete_site_option_{$option}'

Reply:
Ended up going a slightly different route just using the network option directly, but had to disable the multisite checks. `site_options` aren't exclusive to mulitisite installs—just not really used much for single sites.